### ISSUE TITLE
Replacing a picture doesn't work

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/NGT-signbank#271]
* Uploading a first image through the interface works well. 
* In ASL Signbank: Uploading new one leads to an empty slot with a question mark icon ('can't find image'?)
* In NGT Signbank: Uploading a new one throws an error:

KeyError at /video/upload/
'streams'
Request Method:	POST
Request URL:	https://signbank.science.ru.nl/video/upload/
Django Version:	1.6.1
Exception Type:	KeyError
Exception Value:	
'streams'
Exception Location:	/var/www2/signbank/live/virtualenv/signbank/lib/python2.7/site-packages/CNGT_scripts/python/resizeVideos.py in extract_video_properties, line 65
Python Executable:	/usr/bin/python
Python Version:	2.7.6
Python Path:	
['/var/www2/signbank/live/virtualenv/signbank/lib/python2.7/site-packages/pip-8.1.1-py2.7.egg',
 '/var/www2/signbank/live/virtualenv/signbank/lib/python2.7/site-packages',
 '/usr/lib/python2.7',
 '/usr/lib/python2.7/plat-x86_64-linux-gnu',
 '/usr/lib/python2.7/lib-tk',
 '/usr/lib/python2.7/lib-old',
 '/usr/lib/python2.7/lib-dynload',
 '/usr/local/lib/python2.7/dist-packages',
 '/usr/lib/python2.7/dist-packages',
 '/usr/lib/python2.7/dist-packages/PILcompat',
 '/usr/lib/python2.7/dist-packages/gtk-2.0',
 '/usr/lib/pymodules/python2.7']
Server time:	Tue, 7 Feb 2017 10:16:18 +0100

**Comment by Woseseltops** [comment]
@ocrasborn I just spent some time trying to replicate the behavior in both Signbanks, but for me everything works. What Sign and image are you using? Maybe you can show the problem to me live?

The error you show for NGT Signbank makes me think you tried to upload an image in the video field, could that be the case? (if yes: we might want to create an issue to give the user better feedback when this happens :) ).

**Comment by ocrasborn** [comment]
@Woseseltops : I can't replicate the issue for NGT anymore either. I'm sure I used the right button to select an image, but I may have then clicked on the confirm button for the video, which deletes the just selected image. Don't understand how I generated the error above, though.
For ASL, I still don't see anything after I select and upload an image. See my test, http://applejack.science.ru.nl/asl-signbank/dictionary/gloss/2206/?edit. There's a big placeholder rectangle with a small question mark, no image. (And so I cannot evaluate whether the image was in fact replaced or not.)
